src/app/components/SelectOption/index.jsx

```diff
@@ -0,0 +1,54 @@
+import { Select } from 'antd';
+import React, { useState } from 'react';
+
+import './styles.css';
+
+const { Option } = Select;
+
+export default function SelectOption({
+	options,
+	getValueSelected,
+	width,
+	placeholder,
+	defaultValue = ''
+}) {
+	const [isArrowUp, setIsArrowUp] = useState(false);
+
+	const handleChange = value => {
+		getValueSelected && getValueSelected(value);
+		setIsArrowUp(false);
+	};
+
+	return (
+		<div className='select_option-custom' style={{ width: width }}>
+			<Select
+				style={{ width: '100%' }}
+				onChange={handleChange}
+				onFocus={() => setIsArrowUp(true)}
+				onBlur={() => setIsArrowUp(false)}
+				showSearch
```

Comment:
i can not remove it
i need it for my animation

Reply:
Just `Select`.
Because you're using the same-name component *Select* from antd, to avoid duplicate name issue, you can change name of *Select* import from antd by using `import { X as Y } from 'module'` syntax.

For example:
```jsx
import { Select as AntSelect } from 'antd';
```